superset/assets/spec/javascripts/explore/components/ExploreViewContainer_spec.js

```diff
@@ -1,38 +0,0 @@
-// this test must be commented out because ChartContainer is now importing files
```

Comment:
I'm confused as to why this was commented out to start with (?)

Reply:
Looks like it predates the history on github
https://github.com/apache/incubator-superset/commits/master/superset/assets/spec/javascripts/explore/components/ExploreViewContainer_spec.js
Guess we'll never know... 🤔